Tasks/LambdaInvokeFunction/TaskOperations.ts

```diff
@@ -49,6 +53,17 @@ export class TaskOperations {
         }
     }
 
+    // A quick heuristic for if it is json. In the past, it was possible to put in just a string
+    // which is not valid JSON. This heurstic checks for if it is a JSON string, object, or array
+    private isJson(input: string): boolean {
+        const i = input.trim()
+
+        return (
+            (i.startsWith('"') || i.startsWith('{') || i.startsWith('[')) &&
```

Comment:
this would match `{ "hello"` do we want to allow that? or do we need it to be :

```
startsWith('"') && endsWith('"') || 
    startsWith('{') && endsWith('}') || 
    startsWith('['] && endsWith(']')
```

Reply:
So what's the error that you'd get if you tried that now? A rejection from the server?

Reply:
The normal 
```
InvalidRequestContentException: Could not parse request body into json: Unexpected end-of-input: expected close marker for Object
```